observability/src/main/java/io/grpc/observability/GlobalLoggingTags.java

```diff
@@ -51,11 +51,11 @@
   }
 
   Map<String, String> getLocationTags() {
-    return locationTags;
+    return ImmutableMap.<String, String>builder().putAll(locationTags).build();
```

Comment:
`locationTags` was already an `ImmutableMap` (even if declared as just a Map). What did you achieve by creating another ImmutableMap copy? If you want you can change the type of `this.locationTags` and `this.customTags` to `ImmutableMap` so this kind of misunderstanding doesn't happen

Reply:
This was done by the IDE automatically, I will undo this change.